### ISSUE TITLE
Many false positives when using blocklist

### ISSUE THREAD

**Issue by romaincointepas** [feross/webtorrent#863]
Getting many peers being blocked (`blockedPeer` event) even if blacklist doesn't include those peers IPs.

I did a manual check of the blocklist (trying to find IPs or containing ranges) and a test using [PeerBlock](http://forums.peerblock.com/read.php?1,15390) while trying to ping those IPs and in both cases IPs were not there or blocked.

_Note: I could also find IPs that were blocked (`blockedPeer`) that appeared in the blocklist and were blocked by Peerblock._

One can reproduce with the following IPs:

```
83.249.18.137
61.84.203.79
101.50.115.240
58.224.143.38
```

And this blocklist file:
https://gist.githubusercontent.com/anonymous/aeafadfc4af0606fb36a585dca0f92ff/raw/6eca1bd1f193023d0797cc694393c29f3417f016/gistfile1.txt

If this is indeed a bug, it is probably coming from the [fisch0920/ip-set](https://github.com/fisch0920/ip-set) module, but as there doesn't seem to be any activity there I thought it would be better to create the issue here.

---
- WebTorrent version: 0.95.2

**Comment by feross** [comment]
@romaincointepas Can you produce a smaller test case? That blocklist contains a huge amount of IPs which makes it hard to confirm that the 4 IPs you listed are not included in the list.

**Comment by Atavic** [comment]
That's IBlocklist Corporate List.

**Comment by feross** [comment]
I'm closing this issue because it's been a while without a response from the issue opener. It's not possible to debug this issue without additional information. If this is still an issue, please leave a comment or open a new issue.
